javascript/javascript1/week4/index.js

```diff
@@ -0,0 +1,115 @@
+    }
+          
+   if(command.includes('What is my name')){
+    if (nameArray!="")
+    console.log(`Your name is ${nameArray[0]}`)
+  else 
+    console.log("Your name has not yet been mentioned") 
+    
+   }  
+   
+   
+    if (command.includes('Add fishing to my todo')){
+     
+        todoName.push('fishing')
+        console.log('fishing added to your todo')              
+    }else if (command.includes('Add singing in the shower to my todo')){
+     
+        todoName.push('singing in the shower')
+        console.log('singing in the shower added to your todo')              
+    }
+
+    if(command.includes('Remove fishing from my todo'))
+    {
+        todoName.shift()
+        console.log('fishing remove from your todo list')
+    }
+      
+    if (command.includes('what is on my todo')){
+
+        console.log(`You have ${todoName.length} item(s) in your todo list: ${todoName}`);
+    }
+
+    if(command.includes('What day is it today?')){
+        const monthNames = ["January", "February", "March", "April", "May", "June",
+                           "July", "August", "September", "October", "November", "December"]
+        const date = new Date()
+       
+        const ye = new Intl.DateTimeFormat('en', { year: 'numeric' }).format(date)
+        const mo = monthNames[date.getMonth()]//new Intl.DateTimeFormat('en', { month: 'short' }).format(date)
+        const da = new Intl.DateTimeFormat('en', { day: '2-digit' }).format(date)
+
+       console.log(`${da}-${mo}-${ye}`)
+    }
+    
+   
+    let opratorSplitArray=[]
+    let oprator
+    if (command.includes('+')|| command.includes('-') ||command.includes('*')||command.includes('/')){
+        opratorSplitArray=command.split(' ')
+       
+       if(command.includes('+'))
+       {
+         oprator=Number(opratorSplitArray[2])+ Number(opratorSplitArray[4])
+        console.log(`${opratorSplitArray[2]} + ${opratorSplitArray[4]} = ${oprator}`)
+       }
+       if(command.includes('-'))
+       {
+        oprator=Number(opratorSplitArray[2])- Number(opratorSplitArray[4])
+        console.log(`${opratorSplitArray[2]} - ${opratorSplitArray[4]} = ${oprator}`)
+       }
+       if(command.includes('*'))
+       {
+        oprator=Number(opratorSplitArray[2])* Number(opratorSplitArray[4])
+        console.log(`${opratorSplitArray[2]} * ${opratorSplitArray[4]} = ${oprator}`)
+       }
+       if(command.includes('/'))
+       {
+        oprator=Number(opratorSplitArray[2])/ Number(opratorSplitArray[4])
+        console.log(`${opratorSplitArray[2]} / ${opratorSplitArray[4]} = ${oprator}`)
+       }
+
+       
+        
+    }
+
+    if (command.includes('timer')){           
+        let timerSplitArray=[]
+        timerSplitArray=command.split(' ')
+        console.log(`Timer set for ${timerSplitArray[4]} minutes`)
+        
+        function setTimer(){
+            console.log('Timer done')
+        }
+        setTimeout(setTimer, 60000,  );
```

Comment:
it's 6 min, if you want to use the value from timerSplitArray then  
` setTimeout(setTimer,timerSplitArray[4]*1000  );